pandas/core/generic.py

```diff
@@ -771,9 +771,17 @@ def swapaxes(self, axis1: Axis, axis2: Axis, copy: bool_t | None = None) -> Self
         """
         Interchange axes and swap values axes appropriately.
 
+        .. deprecated:: 2.1.0
+                 ``swapaxes`` will be deprecated.
+                 Please use ``transpose`` instead.
```

Comment:
it is deprecated - can we say "will be removed"?

also, the indentation looks like it has an extra level (should just be 4 spaces?)

Reply:
Done.